recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   skip: true  # [py<36]
```

Comment:
python 3.6 is not supported anymore: https://github.com/aio-libs/multidict/blob/v6.0.2/setup.py#L63-L67